Cargo.toml

```diff
@@ -26,6 +26,7 @@ homepage = "https://rust.iceberg.apache.org/"
 
 repository = "https://github.com/apache/iceberg-rust"
 license = "Apache-2.0"
+rust-version = "1.75.0"
```

Comment:
Please also update packages's `rust-version` to `workspakce = true`

Reply:
Hi there! Could you please provide more details or clarify where exactly you are suggesting to add this setting?